common/models/user.js

```diff
@@ -402,6 +408,20 @@ module.exports = function(User) {
   };
 
   /**
+   * A default verification token generator which accepts a callback function.
+   * This one uses the crypto library and 64 random bytes (converted to hex)
+   * for the token. When used in combination with the user.verify() method this
+   * function will be called with the `user` object as it's context (`this`).
+   *
+   * @options {Function} tokenCallback The generator will pass back the new token with this function call
+   */
+  User.generateVerificationToken = function(tokenCallback) {
+    crypto.randomBytes(64, function(err, buf) {
+      tokenCallback(err || buf.toString('hex'));
```

Comment:
Good call, my bad. The name was because we already had `cb` in the other function. Thanks!